### PR TITLE
Tag madness

### PR DESCRIPTION
**Pull request by briannons**
minor refactors.

Eventually this branch is for the scannable string development.

### CODE REVIEW

File: app/src/main/java/net/geekinpurple/www/appofourown/WorkInstance.java [hunk 1]
```diff
@@ -63,4 +66,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
```
Comment (darkmane) [inline app/src/main/java/net/geekinpurple/www/appofourown/WorkInstance.java:69]: Dead code.

I know you're probably planning on using it later, but for now it's dead code.

File: app/src/main/java/net/geekinpurple/www/appofourown/Work.java [hunk 2]
```diff
@@ -142,8 +136,17 @@ public Work(Element work) {
             this.summary = "";
         }
         Log.d("testing", this.summary);
+
+        //Parse for work's Tags
+        Elements ships = work.getElementsByClass("relationships");
```
Comment (darkmane) [inline app/src/main/java/net/geekinpurple/www/appofourown/Work.java:141]: Loop does nothing, and ships makes me think of starships or aircraft carriers not relationships. Slang can be problematic in code.

File: app/src/main/java/net/geekinpurple/www/appofourown/Work.java [hunk 3]
```diff
@@ -113,24 +115,16 @@ public Work(Element work) {
         Element requiredTags = work.getElementsByClass("required-tags").get(0);
 
         // Parse for work's Rating
-        Element rating = requiredTags.getElementsByClass("rating").get(0);
-        String rate = parseClass(rating);
-        this.rating = Rating.valueOf(rate);
+        this.rating = parseTag(Rating.class, requiredTags, "rating");
```
Comment (darkmane) [inline app/src/main/java/net/geekinpurple/www/appofourown/Work.java:118]: Probably want to make "rating", "warnings", etc constants.

**Comment by briannons** [comment]
ships is the fandom accepted term for relationships?

**Pull request merged**